.github/test-a-feature.md

```diff
@@ -51,9 +51,9 @@ These tests are typically imported into individual component tests.
 Every common test receives your component as its first argument.
 
 ```tsx
-import { isConformant } from 'test/specs/commonTests'
+import { isConformant } from '../../commonTests'
 
-import Divider from 'src/components/Divider/Divider'
+import Divider from '@fluentui/react/src/components/Divider/Divider'
```

Comment:
```suggestion
import { Divider } from '@fluentui/react'
```

Should it be just this?

Reply:
Not sure, I wondered if the original deep import was intentional to reduce the amount Jest had to parse. But since in this project it will eventually have to parse pretty much everything in src eventually, it might not matter.

Reply:
Not, it's not intentional, this pattern comes from SUIR

Reply:
Okay, removed the deep imports where possible